scripts/fetch/fetch.precip_shapefiles.R

```diff
@@ -1,9 +1,60 @@
+# Download all available shapefiles in our date range
+fetch.precip_shapefiles <- function(viz = as.viz('precip_shapefiles')){
```

Comment:
yes, this is nice. Note I added and `unlink` in my PR that will collide w/ this change. Not sure how we want to do that, but we want the result of this fetch to honor the changes to 
```yaml
id: dates
startDate: "2017-09-18"
endDate: "2017-09-25"
```
specifically, that unlink will deal with the more common issue of moving the startDate forward in time, but should work for moving endDate back too. (the dir had stale precip from 9/12 and forward, even with this date specified).

Reply:
Note that just because we have a file it does not mean it won't change. This data gets QA/QCd as it ages and the files can update. So per file time stamp checking is needed or we need to clean and rebuild frequently to be proper.